### PR TITLE
spencer/eks aggregator

### PR DESCRIPTION
**Pull request by spencergilbert**

### CODE REVIEW

File: eks-aggregator/helm/datadog.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ agents:
   useConfigMap: true
   customAgentConfig:
     logs_config:
-      logs_dd_url: "vector-haproxy.vector.svc.cluster.local:8080"
+      logs_dd_url: "internal-k8s-vector-vector-e67b4af534-965797666.us-east-1.elb.amazonaws.com:8080"
```
Comment (jdrouet) [inline eks-aggregator/helm/datadog.yaml]: 🤔 sounds really specific, won't that change after being redeployed?

File: eks-aggregator/http-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internal
+    alb.ingress.kubernetes.io/healthcheck-port: '8686'
+    alb.ingress.kubernetes.io/healthcheck-path: /health
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 8080}]'
+    alb.ingress.kubernetes.io/target-type: ip
+    kubernetes.io/ingress.class: alb
+spec:
+  rules:
+    - host: internal-k8s-vector-vector-e67b4af534-965797666.us-east-1.elb.amazonaws.com
```
Comment (jdrouet) [inline eks-aggregator/http-ingress.yaml]: same here?

File: eks-aggregator/helm/datadog.yaml (hunk 1 above)
Comment (spencergilbert) [reply eks-aggregator/helm/datadog.yaml]: Looking at the most recent commit, that's a generic hostname intended to be replaced with the `--set` option 🤔

File: eks-aggregator/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+```shell
+kubectl --namespace vector get ingress vector \
+	--output go-template='{{(index .status.loadBalancer.ingress 0 ).hostname}}'
+```
+
+Then install your Datadog Agents substituting the hostname from the previous step.
+
+```shell
+helm upgrade --install datadog datadog/dataodg \
```
Comment (jszwedko) [inline eks-aggregator/README.md:49]:
```suggestion
helm upgrade --install datadog datadog/datadog \
```

**Review by jszwedko — APPROVED** [review]
🚀 I was able to run the demo.

**Pull request merged**